.github/workflows/lint.yml

```diff
@@ -0,0 +1,42 @@
+# Run linters automatically on pull requests.
+name: Lint
+on:
+  pull_request:
+
+jobs:
+  ament_lint:
+    name: ament_${{ matrix.linter }}(${{ matrix.ros_distribution }})
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ matrix.docker_image }}
+    strategy:
+      fail-fast: false
+      matrix:
+        ros_distribution:
+          - foxy
+          - galactic
+          - rolling
+        include:
+          # Foxy Fitzroy (May 2020 - May 2023)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-foxy-ros-base-latest
+            ros_distribution: foxy
+            ros_version: 2
+          # Galactic Geochelone (May 2021 - November 2022)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-galactic-ros-base-latest
+            ros_distribution: galactic
+            ros_version: 2
+          # Rolling Ridley  (June 2020 - Present)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-rolling-ros-base-latest
+            ros_distribution: rolling
+            ros_version: 2
+        linter:
+          - copyright
+          - xmllint
+          - lint_cmake
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - uses: ros-tooling/action-ros-lint@0.1.3
```

Comment:
Just out of curiosity, why did you add this? Is `action-ros-ci` not enough?

Reply:
To clarify which linter fails and to use minimum lint checks.

Reply:
> To clarify which linter fails

I see, so do you mean "We'll struggle to find the error reasons if we use `action-ros-ci`, but with `action-ros-lint` it's easier to understand.", right?

> to use minimum lint checks

For example, what checks do you think are unnecessary for PR checks?

Reply:
> I see, so do you mean "We'll struggle to find the error reasons if we use action-ros-ci, but with action-ros-lint it's easier to understand.", right?

Yes!

Reply:
> For example, what checks do you think are unnecessary for PR checks?

I think c++ and python linters are not necessary for vendor packages because external non-ros packages may use different formats.

Reply:
I understand, but if that's the case, is there no problem with `action-ros-ci` in the other workflow?
https://github.com/tier4/osqp_vendor/pull/9/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R34

So I mean that if you don't check C++/Python linters here, they will be checked in the other workflow and I guess the CI fails.
Do you plan to set it `Required` only for `lint.yml` and won't set `build.yml` `Required`?

Reply:
I do not add `test` into `colcon default`, so test won't be executed in `build` workflow.

Reply:
Hmm, but it seems to be executed. :thinking: 
https://github.com/tier4/osqp_vendor/runs/4404280889?check_suite_focus=true

![image](https://user-images.githubusercontent.com/31987104/144747986-1e0ef4c1-892f-4ae9-817a-934c74cc80a4.png)

Reply:
I skipped tests at https://github.com/tier4/osqp_vendor/pull/9/commits/06629b3bd82c1d7af50e46ed83bc1f52ff0a0484.

Reply:
Thank you!
btw, I can't resolve comments. :smiling_face_with_tear: 
![image](https://user-images.githubusercontent.com/31987104/144753272-55ee6daf-cc6f-4604-91a3-bd03174edeb4.png)